draft-birkholz-rats-corim.md

```diff
@@ -178,6 +178,42 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
 
 Privacy Considerations
 
+# Concise SWID Data Extension
+
+# 2.7 The link-entry Map
+
+The link-entry map has been extended to define new relationship between a
+CoSWID and CoMID
+
+When defining CoSWID to CoMID link entry map the following optional elements
+should not be used:
+* artifact
+* ownership
+* use
+
+The new rel values (as listed section 4.4) will not be defined in ISO/IEC 19770-2:2015
+[SWID] specification. Hence for modelling CoSWID to CoMID relations
+ISO/IEC 19770-2:2015 [SWID] specification shall not be used.
+
+# CoSWID Indexed Label Values
+
+# 4.4. Link Rel Values
+
+The Link Rel values as mentioned in Table 6 of CoSWID specification
+has been extended to add new indexes which defines the relationship
+between a CoSWID to CoMID.
+
+ | Index | Relationship Type Name | Definition                                     |
+ --------|----------------------- |--------------------------------------------|
+ | 12    | m-requires             | The link references a prerequisite module  |
+ |       |                        | tag(s) to be loaded or present             |
+ |       |                        | for installing this software.       |      |
+ | 13    | m-runs-on              | The link references a module tag that      |
+ |       |                        | this software run on                       |
+
+New indexes 12 and 13 will be registered in the
+IANA "Software Tag Link Relationship Values" registry defined in Section 6.2.7.
```

Comment:
hardcoded references to section (i.e., "Section 6.2.7.") is going to break.  You need to add an anchor in the referenced section and put a link here to that.